src/Raven.Client/Documents/Operations/Revisions/GetRevisionsOperation.cs

```diff
@@ -0,0 +1,77 @@
+//-----------------------------------------------------------------------
+// <copyright file="RevisionsConfiguration.cs" company="Hibernating Rhinos LTD">
+//     Copyright (c) Hibernating Rhinos LTD. All rights reserved.
+// </copyright>
+//-----------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Net.Http;
+using Raven.Client.Documents.Commands;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Extensions;
+using Raven.Client.Http;
+using Raven.Client.Json.Serialization.NewtonsoftJson.Internal;
+using Sparrow.Json;
+
+namespace Raven.Client.Documents.Operations.Revisions
+{
+    public class GetRevisionsOperation<T> : IOperation<RevisionsResult<T>>
+    {
+        private readonly GetRevisionsResultCommand _command;
+
+        public GetRevisionsOperation(string id, int? start = 0, int? pageSize = int.MaxValue, bool metadataOnly = false)
+        {
+            _command = new GetRevisionsResultCommand(id, start, pageSize, metadataOnly);
+        }
+
+        public RavenCommand<RevisionsResult<T>> GetCommand(IDocumentStore store, DocumentConventions conventions, JsonOperationContext context, HttpCache cache)
+        {
+            return _command;
```

Comment:
I do not recall a single command returning always the same instance. What are the implications? I think it would be better to just create new instance here.

Reply:
Yes